### ISSUE TITLE
EPI Cheatsheets

### ISSUE THREAD

**Issue by dirkschumacher** [R4EPI/projmgmt#9]
Just an idea. Have some RStudio like cheatsheets for specific topics. Maybe stata <-> R or general concepts for epi (e.g. how to do an epi curve).

https://www.rstudio.com/resources/cheatsheets/

**Comment by aspina7** [comment]
Don't think Stata r necessary but an epi one would be great!

**Comment by thibautjombart** [comment]
Yes, actually that's something I had wanted e.g. for RECON packages for a while. Threw the idea of templated/automated cheathsheets for a rOpenSci hackathon, I think something came out of it.. oh, dassit:
https://gitpitch.com/ramnathv/rcheatsheet/master?grs=github&t=moon
